views/repeated-metas.php

```diff
@@ -8,15 +8,19 @@
  * @license      GPL-2.0-or-later
  */
 
-?>
-<meta name="parsely-title" content="<?php echo esc_attr( $parsely_page['headline'] ); ?>" />
-<meta name="parsely-link" content="<?php echo esc_attr( $parsely_page['url'] ); ?>" />
-<meta name="parsely-type" content="<?php echo esc_attr( $parsely_post_type ); ?>" />
-<meta name="parsely-image-url" content="<?php echo esc_attr( $parsely_page['thumbnailUrl'] ); ?>" />
-<meta name="parsely-pub-date" content="<?php echo esc_attr( $parsely_page['datePublished'] ); ?>" />
-<meta name="parsely-section" content="<?php echo esc_attr( $parsely_page['articleSection'] ); ?>" />
-<meta name="parsely-tags" content="<?php echo esc_attr( $parsely_page['keywords'] ); ?>" />
-<?php foreach ( (array) $parsely_page['author'] as $parsely_author ) { ?>
+foreach ( $parsely_metas as $parsely_meta_key => $parsely_meta_val ) {
+	if ( ! empty( $parsely_meta_val ) && is_string( $parsely_meta_val ) ) {
+		echo '<meta name="parsely-' . esc_attr( $parsely_meta_key ) . '" content="' . esc_attr( $parsely_meta_val ) . '" />' . "\n";
```

Comment:
Rather than a concatenated string of 6 pieces, it's probably easier to see the format of the HTML being added, and that the values are correctly escaped, if something like the following is used (untested):

```suggestion
		printf(
			'<meta name="%s" content="%s" />',
			esc_attr( 'parsely-' . $parsely_meta_key ),
			esc_attr( $parsely_meta_val )
		) . "\n";
```